[RI1 Ch3] Multiple vCPU corrections (#2099)
cntt-n/CNTT: doc/ref_impl/cntt-ri/chapters/chapter03.md

| Crypto acceleration | N  | Y | Y |
| Crypto Acceleration Interface | N  | Y | Y |--->

| .conf | Basic | Network Intensive |
|-------|----------------|----------------|
| CPU allocation ratio  | 4:1 | 1:1  |
| NUMA awareness | N | Y |
| CPU pinning capability | N | Y |
| Huge Pages | N | Y |
| Catalogue storage Types | Y  | Y  |
| Storage Block | Y | Y |
| Storage Object | Y | Y |
| Storage with replication | N | Y |
| Storage with encryption | Y | Y |
| Storage IOPS oriented | N | Y |
| Storage capacity oriented | N | N |
| vNIC interface | virtio1.1 |  virtio1.1|
| Overlay protocol | VXLAN, MPLSoUDP, GENEVE, other |  VXLAN, MPLSoUDP, GENEVE, other |
| NAT | Y | Y |
| Security Group | Y | Y |
| SFC support | N | Y |
| Traffic patterns symmetry | Y | Y |
| vSwitch optimisation | N | Y, DPDK |
| Support of HW offload | N | Y, support of SmartNic |
| Crypto acceleration | N  | Y |
| Crypto Acceleration Interface | N  | Y |

<a name="3.4"></a>
## 3.4 NFVI Hardware Profile

> [RM1: 5.4 NFVI HW profiles features and requirements](../../../ref_model/chapters/chapter05.md#5.4) defines the NFVI hardware layer profiles.The labs are typically provisioned with the minimal required hardware and thus it is difficult to partition the available hardware to provision/configure multiple NFVI profiles. However, when reference implementations and the follow up testing and verification are conducted, the hardware profile need to be clearly described. This is especially important for performance testing and verification.

Reviewer: The Section heading has been changed and also we have started using "Cloud infrastructure" instead of NFVI -- sorry, should have pointed out earlier.

```suggestion
[RM1: 5.4 Cloud Infrastructure Hardware Profiles features and requirements](../../../ref_model/chapters/chapter05.md#5.4) defines the Cloud Infrastructure hardware layer profiles.The labs are typically provisioned with the minimal required hardware and thus it is difficult to partition the available hardware to provision/configure multiple NFVI profiles. However, when reference implementations and the follow up testing and verification are conducted, the hardware profile need to be clearly described. This is especially important for performance testing and verification.
```

| nfvi.hw.cpu.cfg.003 | NUMA | NUMA support and BIOS configured to enable NUMA | N | Y | Y |
| nfvi.hw.cpu.cfg.004 | Simultaneous Multithreading (SMT) | This permits multiple independent threads of execution on a single core. | Y | Y|
| nfvi.hw.cac.cfg.001 | GPU | GPU | N | N |
| nfvi.hw.stg.hdd.cfg.001* | Local Storage HDD | Hard Disk Drive |  |  |
| nfvi.hw.stg.ssd.cfg.002* | Local Storage SSD | Solid State Drive | Recommended | Recommended |
| nfvi.hw.nic.cfg.001 | NIC Ports | Total Number of NIC Ports available in the host | 4 | 4 |
| nfvi.hw.nic.cfg.002 | Port Speed | Port speed specified in Gbps (minimum values) | 10 | 25 |
| nfvi.hw.pci.cfg.001 | PCIe slots | Number of PCIe slots available in the host | 8 | 8 |
| nfvi.hw.pci.cfg.002 | PCIe speed |  | Gen 3 | Gen 3 |
| nfvi.hw.pci.cfg.003 | PCIe Lanes |  | 8 | 8 |
| nfvi.hw.bdc.cfg.001 | Bonded VLAN ports |  | Y | Y |
| nfvi.hw.nac.cfg.001 | Cryptographic Acceleration | IPSec, Crypto |  N | Optional |
| nfvi.hw.nac.cfg.002 | SmartNIC | A SmartNIC that is used to offload network functionality to hardware | N | Optional  |
| nfvi.hw.nac.cfg.003 | Compression |  |  |

> ## 3.5 NFVI Required State

Reviewer: ```suggestion
## 3.5 Cloud Infrastructure Required State
```

> This sections describes the readiness of NFVI before the certification process can begin. Once the NFVI is configured with either of the profiles - B, <!---C,---> N, a set of tests (for example functests) should be run in order to determine the readiness of NFVI for certification.

Reviewer: ```suggestion
This sections describes the readiness of Cloud Infrastructure before the certification process can begin. Once the Cloud Infrastructure is configured with either of the profiles - B, <!---C,---> N, a set of tests (for example functests) should be run in order to determine the readiness of the Cloud Infrastructure for certification.
```

| `req.inf.com.01` | **must** provide compute resources for VM instances.  |
| `req.inf.com.02` | **should** include industry standard hardware management systems at both HW device and platform level |
| `req.inf.com.03` | **should** support symmetrical CPU multi-processing with shared memory access as well as multi-threading. |

> | `req.inf.com.04` | **must** be able to support multiple CPU types to support Base, Network Intensive<!---, and Compute Intensive---> infrastructure profiles. |

Reviewer: Please see comment made in PR #2097

Author: @pgoyal01, we have added the term to the glossary. Should we still define it here as well?

| Orchestration: Heat | https://docs.openstack.org/api-ref/orchestration/v1/index.html#api-versions  | v1.0 | NA |
| Acceleration: Cyborg | https://docs.openstack.org/cyborg/pike/userdoc/api.html | v1.0 | NA |

> ## 3.6 NFVI and VIM Architecture

Reviewer: ```suggestion
## 3.6 Cloud Infrastructure and VIM Architecture
```

> This sections concludes the expectation for NFVi and VIM architecture according to [RA1: Chapter 3 NFVI + VIM Architecture](../../../ref_arch/openstack/chapters/chapter03.md)

Reviewer: To be consistent with the use of NFVI by CNTT

```suggestion
This sections concludes the expectation for NFVI and VIM architecture according to [RA1: Chapter 3 NFVI + VIM Architecture](../../../ref_arch/openstack/chapters/chapter03.md)
```

> This sections concludes the expectation for NFVI and VIM architecture according to [RA1: Chapter 3 NFVI + VIM Architecture](../../../ref_arch/openstack/chapters/chapter03.md)

Reviewer: ```suggestion
This sections concludes the expectation for Cloud Infrastructure and VIM architecture according to [RA1: Chapter 3 Cloud Infrastructure + VIM Architecture](../../../ref_arch/openstack/chapters/chapter03.md)
```

|Cloud Controller Services|All components must be deployed within a high available architecture that can withstand at least a single node failure and respects the anti-affinity rules for the location of the services|
|Physical Network|The recommended network architecture is spine and leaf topology; however, for small sites, a legacy topology (access/aggregation switches) can be set up.

> ## 3.7 NFVI and VIM Component Level Architecture

Reviewer: ```suggestion
## 3.7 Cloud Infrastructure and VIM Component Level Architecture
```

> This sections concludes the expectation for NFVi and VIM component level architecture according to [RA1: Chapter 4 NFVI + VIM Component Level Architecture](../../../ref_arch/openstack/chapters/chapter04.md)

Reviewer: To be consistent with the use of NFVI by CNTT

```suggestion
This sections concludes the expectation for NFVI and VIM component level architecture according to [RA1: Chapter 4 NFVI + VIM Component Level Architecture](../../../ref_arch/openstack/chapters/chapter04.md)
```

> This sections concludes the expectation for NFVI and VIM component level architecture according to [RA1: Chapter 4 NFVI + VIM Component Level Architecture](../../../ref_arch/openstack/chapters/chapter04.md)

Reviewer: ```suggestion
This sections concludes the expectation for Cloud Infrastructure and VIM component level architecture according to [RA1: Chapter 4 Cloud Infrastructure + VIM Component Level Architecture](../../../ref_arch/openstack/chapters/chapter04.md)
```
